api/types/src/convert.rs

```diff
@@ -1082,6 +1007,72 @@ impl<'a, S: StateView> MoveConverter<'a, S> {
         let id = code.identifier_at(func.name);
         Ok(id.to_string())
     }
+
+    fn try_to_entry_function_payload(&self, fun: EntryFunction) -> Result<EntryFunctionPayload> {
+        let (module, function, ty_args, args) = fun.into_inner();
+        let func_args = self
+            .inner
+            .view_function_arguments(&module, &function, &ty_args, &args);
+
+        let json_args = match func_args {
+            Ok(values) => values
+                .into_iter()
+                .map(|v| MoveValue::try_from(v)?.json())
+                .collect::<Result<_>>()?,
+            Err(_e) => args
+                .into_iter()
+                .map(|arg| HexEncodedBytes::from(arg).json())
+                .collect::<Result<_>>()?,
+        };
+
+        Ok(EntryFunctionPayload {
+            arguments: json_args,
+            function: EntryFunctionId {
+                module: module.into(),
+                name: function.into(),
+            },
+            type_arguments: ty_args.into_iter().map(|arg| arg.into()).collect(),
+        })
+    }
+
+    fn try_to_aptos_core_entry_function(
```

Comment:
If the function name `try_into_entry_function` doesn't already exist, perhaps it may be used.

```suggestion
    fn try_into_entry_function(
```

Reply:
They are 2 different functions one converts `api::EntryFucntion->core::EntryFucntion` and the other does the reverse conversion.